support/cas-server-support-thymeleaf/src/main/resources/templates/fragments/pwdupdateform.html

```diff
@@ -53,6 +53,32 @@
                 <span th:each="err : ${#fields.errors('*')}" th:utext="${err}">Error text</span>
             </div>
 
+            <div id="currentPasswordInputForm"
+                 class="cas-field form-group mdc-input-group my-3"
+                 th:if="${activeFlowId} == 'login'">
```

Comment:
Just to be sure: does this work for cases where the user has forgotten their password?